static/js/shared/constants.ts

```diff
@@ -37,6 +37,11 @@ export const EARTH_NAMED_TYPED_PLACE: NamedTypedPlace = {
   name: "Earth",
   types: ["Planet"],
 };
+export const USA: NamedTypedPlace = {
```

Comment:
for consistency with the other NamedTypedPlace constants, let's name this "USA_NAMED_TYPED_PLACE"

Reply:
Changed the name to USA_NAMED_TYPED_PLACE, for consistency!